src/Faker/Provider/ar_MA/Company.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Faker\Provider\ar_MA;
+
+class Company extends \Faker\Provider\Company
+{
+    protected static $formats = array(
+        '{{lastName}} {{companySuffix}}',
+        '{{companyPrefix}} {{lastName}} {{companySuffix}}',
+        '{{companyPrefix}} {{lastName}}',
+    );
+
+    protected static $bsWords = array(
+        array()
+    );
+
+    protected static $catchPhraseWords = array(
```

Comment:
The dataset is too small, the generated catch phrases will all look the same. Can you add more data?

Reply:
I will